samples/net/zperf/src/zperf_internal.h

```diff
@@ -67,7 +67,7 @@ struct zperf_udp_datagram {
 	s32_t id;
```

Comment:
Please change the commit message a bit. So instead of `Please refer to issues #15605.` say `Fixes #15605`. That way the issue will be automatically closed when this PR is merged.

Reply:
> Please change the commit message a bit. So instead of `Please refer to issues #15605.` say `Fixes #15605`. That way the issue will be automatically closed when this PR is merged.

It's good way. Done.
Thanks.